### ISSUE TITLE
cilium-bugtool: "writing file: unexpected EOF"

### ISSUE THREAD

**Issue by borkmann** [cilium/cilium#18568]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

ciliuum sysdump throws errors such as:
```
⚠️ Failed to unarchive 'cilium-bugtool' output for "cilium-6v2vw": reading file in tar archive: /tmp/2650086882/cilium-sysdump-20220121-090458/cilium-bugtool-cilium-6v2vw-20220121-090458/cmd/state/templates/4d313d5d74172657c0b1d2a63421da83c9d558241053f66d7a487c016935e5f8/bpf_host.asm: writing file: unexpected EOF
```

### Cilium Version

Seen on 1.10 .

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tklauser** [comment]
I think this should be resolved by https://github.com/cilium/cilium-cli/pull/671.

**Comment by tklauser** [comment]
This is a duplicate of cilium/cilium-cli#654 which has been addressed in cilium/cilium-cli#671 and should be fixed the just released [cilium-cli v0.10.1](https://github.com/cilium/cilium-cli/releases/tag/v0.10.1).
